src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -209,7 +209,10 @@ export class QueryResultsView extends Disposable {
 	}
 
 	private setQueryRunner(runner: QueryRunner) {
-		const activeTab = this._input.state.activeTab;
+		if (!this.input || !this.input.state || !this.input.state.visibleTabs) {
```

Comment:
I generally don't think it's correct just to always return immediately if we don't have the required things. You should be figuring out when that scenario would happen and why - in most cases I'd guess that it's unexpected and probably means something went wrong so we should be logging that and likely throwing or something to make sure to bubble these things up.